### PR TITLE
Usage stuff

### PR DESCRIPTION
**Pull request by memosys**
tackles 

#72 (some UX discussion required)
#83  (partially)

want to merge early!

### CODE REVIEW

File: web-portal/backend/src/usage/usage.service.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    const q = `gateway_relay_usage{tenant="${tenantId}"}`;
+
+    console.log({step, stepBack, start, end, q})
+
+    const result = await this.fetchData(q, start, end, step);
+    return result.json();
+  }
+
+  private async fetchData(query: string, start: number, end: number, step: number | string): Promise<Response> {
+    const url = `https://api.fly.io/prometheus/porters-staging/api/v1/query_range?query=${query}&start=${start}&end=${end}&step=${step}`;
```
Comment (plor) [inline web-portal/backend/src/usage/usage.service.ts:41]: Definitely want to move this to env var, hardcoding staging is something that will sneaky bite when moving to prod.

File: web-portal/frontend/utils/types.ts
Comment (plor) [inline web-portal/frontend/utils/types.ts:1]: Is your auto formatter closer to getting fixed?

**Review by plor — APPROVED** [review]
Going to merge, just made a couple comments to consider.

File: web-portal/frontend/utils/types.ts
Comment (memosys) [reply web-portal/frontend/utils/types.ts:1]: I reinstalled didn't work! Gotta use some deep cleaner

**Pull request merged**